templates/index.html

```diff
@@ -1,692 +1,425 @@
-    <div class="wrapper equal-height--vertical-divider">
-        <div class="six-col equal-height--vertical-divider__item">
-            <h2>Learn more</h2>
-            <p>Want to know more about snaps, <code class="command-inline">snapd</code>, Snapcraft and Ubuntu Core?</p>
-            <ul class="no-bullets">
-                <li><a href="https://forum.snapcraft.io/" class="external">Forum</a></li>
-                <li><a href="https://rocket.ubuntu.com/channel/snapcraft" class="external">Real-time chat</a></li>
-                <li><a href="http://stackoverflow.com/questions/ask?tags=snapcraft" class="external">Snapcraft on Stack Overflow</a></li>
-            </ul>
+    <div class="p-strip is-deep">
+      <div class="row">
+        <div class="col-6">
+          <h2>Not sure yet?</h2>
         </div>
-        <div class="six-col last-col equal-height--vertical-divider__item">
-            <h2>Contribute</h2>
-            <p>Interested in helping build Snapcraft and expand its reach?</p>
-            <ul class="no-bullets">
-                <li><a href="https://github.com/canonical-websites/snapcraft.io" class="external">Fork this website on GitHub</a></li>
-                <li><a href="https://github.com/snapcore/snapcraft" class="external">Fork snapcraft on GitHub</a></li>
-                <li><a href="https://github.com/snapcore/snapd" class="external">Fork snapd on GitHub</a></li>
-            </ul>
+        <div class="col-6">
+          <ul class="p-list u-no-margin">
+            <li class="p-list__item"><a href="https://docs.snapcraft.io/build-snaps/">Learn more</a> about snaps and Snapcraft</li>
+            <li class="p-list__item"><a href="https://forum.snapcraft.io/categories">Ask on the forum</a> if you have any questions</li>
+          </ul>
         </div>
+      </div>
     </div>
-</section>
-
-
-</div>
 
-<div class="wrapper">
-    <footer class="footer">
-      <div class="footer__social right">
-        <h3 class="footer__heading">Follow us</h3>
-        <ul class="social-list inline-list clearfix">
-          <li class="social-list__item">
-            <a href="https://twitter.com/snapcraftio" class="social-list__item--twitter">Share on Twitter</a>
-          </li>
-          <li class="social-list__item">
-            <a href="https://plus.google.com/+SnapcraftIo" class="social-list__item--google-plus">Share on Google plus</a>
-          </li>
-          <li class="social-list__item">
-            <a href="https://www.facebook.com/snapcraftio" class="social-list__item--facebook">Share on Facebook</a>
-          </li>
-          <li class="social-list__item">
-            <a href="https://www.youtube.com/snapcraftio" class="social-list__item--youtube">Share on YouTube</a>
-          </li>
-        </ul>
+    <footer class="p-footer p-strip--light">
+      <div class="row">
+        <div class="col-9">
+          <p>&copy; 2017 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
```

Comment:
Try and update the year with a template variable as we are nearing the end of year :)